library/pod-security-policy/users/src.rego

```diff
@@ -0,0 +1,27 @@
+package k8spspallowedusers
+
+violation[{"msg": msg}] {
+  rule := input.parameters.runAsUser.rule
+  input_containers[input_container]
+  provided_user := input_container.securityContext.runAsUser
```

Comment:
@ritazh @ctab what happens here when the securityContext is an empty object {} does this work ok?

Reply:
Looks like it will also need to handle setting securityContext at the Pod Level but being overridden at the container level?

Reply:
Thanks for catching this John. I just pushed some changes to handle both an empty securityContext and pod level definition.